### PR TITLE
Update README with steps to use Spring

### PR DESCRIPTION
**Pull request by tenczar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# java-base-image
+gi# java-base-image
```
Comment (imikushin) [inline README.md]: `gi` is probably a typo. Please remove

File: README.md [hunk 2]
```diff
@@ -121,3 +123,137 @@ $ dispatch exec --json --input '{"name": "Jon"}' --wait hello
...
+```java
+    @Bean
+    BiFunction function(@Qualifier("noone") Person noone) {
+        return new HelloSpringFunction(noone);
+    }
+```
+
+To create this function we run
+```bash
+dispatch create function java8-spring spring-fn ./HelloSpring.class
```
Comment (imikushin) [inline README.md]: Do you mean `./HelloSpring.java`?

**Review by imikushin — CHANGES_REQUESTED** [review]
Awesome! 
Just a couple typos and we're good to go 🚀

File: README.md (hunk 1 above)
Comment (tenczar) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (tenczar) [reply README.md]: done

**Review by imikushin — APPROVED** [review]

**Pull request merged**